content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,141 @@
+
+.. math::
+	H^p = K_{13}I^p = K_{13}\frac{H^p_2}{K_{12}},
+
+where :math:`K_{13}` is similar coefficient with :math:`K`, but between body (Loop1) and Rx (Loop3). Eq. :eq:`HsHpratio` can be simplified as
+
+.. math::
+	\frac{H^s}{H^{p}} = -A Q(\alpha),
+
+where :math:`A = \frac{K_{23}M}{K_{13}L}` and response function, :math:`Q = \frac{\alpha^2+\imath \alpha}{1+\alpha^2}`. The amplitude can be written as
+
+.. math::
+	\Big| \frac{H^s}{H^p} \Big|
+	= \frac{K_{23}M}{K_{13}L} \frac{\alpha}{\sqrt{\alpha^2+1}} .
+
+.. note::
+
+	Because the ratio :math:`\frac{K_{23}M}{K_{13}L}` is generally very small, the ratio :math:`\frac{H^s}{H^{p}_3}` is small, regardless of the value of :math:`\alpha`. Often part per million (ppm) is used for the unit of this ratio.
+
+
+The phase of :math:`\frac{H^s}{H^p}`, :math:`\theta_s` will be same as that of :math:`I(\omega)`, hence
+
+.. math::
+	\theta_s = - \frac{\pi}{2} - \phi,
+	:label: phaseHs
+
+where
+
+.. math::
+	\phi=tan^{-1}(\frac{\omega L}{R})=tan^{-1}(\alpha).
+	:label: phaseHs_phi
+
+.. figure:: ./images/PhaseHs.png
+   :align: center
+   :scale: 60%
+   :name: PhaseHs
+
+   Phase diagram of secondary magnetic field (:math:`H^s`).
+
+.. todo::
+	make the arrows showing the primary and secondary magnetic field blue and red (same as your diagrams showing field lines
+
+From above diagram and Eq. :eq:`phaseHs_phi`, it can be seen that:
+
+.. note::
+	- For a very good conductor: :math:`\alpha = \frac{\omega L}{R} \rightarrow \infty` and :math:`\phi \rightarrow \frac{\pi}{2}`. In this case, phase of the secondary field is 180 :math:`^\circ` (:math:`\pi`) behind the primary field
+
+	- For a very poor conductor: :math:`\alpha = \frac{\omega L}{R} \rightarrow 0` and :math:`\phi \rightarrow 0`.  In this case, phase of the secondary field is 90 :math:`^\circ` (:math:`\frac{\pi}{2}`) behind the primary field
+
+Assuming the phase of the primary magnetic field, :math:`\theta_p=0`, its phase lag, :math:`\psi`, can be written as
+
+.. todo::
+	a quick diagram to show one sinusoid lagging the primary
+
+.. math::
+	\psi = \theta_p - \theta_s =\frac{\pi}{2} + \phi,
+	:label: phaseHs_lag
+
+The lag in the phase of :math:`\frac{\pi}{2}` is due to the inductive coupling between Loop1 and Loop2, whereas the additional phase lag :math:`\phi` is determined by the properties of the conductor as an electrical circuit. That is,
+
+.. math::
+	H^s = K_{23}|I(\omega)| cos (\omega t - (\frac{\pi}{2}+\phi))
+
+	    = K_{23}|I(\omega)| sin (\omega t - \phi)
+
+	    = K_{23}|I(\omega)| \right(
+	    cos (\omega t-\frac{\pi}{2}) cos (\phi) + cos (\omega t-\pi) sin (\phi) \left)
+
+The component of :math:`H^s` 180 :math:`^\circ` out of phase with :math:`H^p` is :math:`H^s sin(\phi)`, whereas the component 90 :math:`^\circ` out-ouf-phase is :math:`H^s cos(\phi)`.
+
+In loop-loop frequency domain EM survey,
+
+.. note::
+
+	- the 180 :math:`^\circ` out-of-phase fraction of :math:`H^s` is called the **real** or **in-phase ** component.
```

Comment:
The [travis fail](https://travis-ci.org/ubcgif/em/builds/122930825) is on this line, it is because of the space after `in-phase` and before the `**`. A bit annoying, but all text-emphasis in rst needs to be right next to the word (this way there is a distinction between the start and end of an emphasis 
<img width="885" alt="build 1108 - ubcgif em - travis ci 2016-04-22 10-39-28" src="https://cloud.githubusercontent.com/assets/6361812/14749879/d3b1b34c-0876-11e6-8aea-5987a8f0491d.png">